server/data-migrations/1675368628726-AddAppVersionToDataQueries.ts

```diff
@@ -0,0 +1,67 @@
+import { App } from 'src/entities/app.entity';
+import { AppVersion } from 'src/entities/app_version.entity';
+import { DataQuery } from 'src/entities/data_query.entity';
+import { Organization } from 'src/entities/organization.entity';
+import { EntityManager, MigrationInterface, QueryRunner, TableColumn, TableForeignKey } from 'typeorm';
+
+export class AddAppVersionToDataQueries1675368628727 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumn(
+      'data_queries',
+      new TableColumn({
+        name: 'app_version_id',
+        type: 'uuid',
+        isNullable: true,
+      })
+    );
+
+    await queryRunner.createForeignKey(
+      'data_queries',
+      new TableForeignKey({
+        columnNames: ['app_version_id'],
+        referencedColumnNames: ['id'],
+        referencedTableName: 'app_versions',
+      })
+    );
+
+    const entityManager = queryRunner.manager;
+    const organizations = await entityManager.find(Organization, {
+      select: ['id', 'name'],
+    });
+
```

Comment:
This can be done
- get distinct of app_version_ids - A
- get a map query id vs app_version_id - B
- iterate A and get query ids under specific app_version_id from B - C
- update all query ids in C with specific app_version_id

A - select distinct(data_sources.app_version_id) from data_queries inner join data_sources on data_queries.data_source_id = data_sources.id
B - select data_queries.id, data_sources.app_version_id from data_queries inner join data_sources on data_queries.data_source_id = data_sources.id